README.md

```diff
@@ -1 +1,62 @@
-# -Global-Warming
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
Good job on recording the video and sending it on the additional information in the submission but kindly add a link to the video on the PR description or README file.